README.md

```diff
@@ -9,141 +9,173 @@ entries to a Giganto server.
 
 * Giganto 0.21.0.
 
-## Build
+## Usage
 
-REproduce may be built as a single binary for portability. Make sure that the
-musl target is available. The following command installs the Rust compiler for
-`x86_64-unknown-linux-musl` if it is not already installed:
+To start REproduce, provide a path to the TOML configuration file:
 
 ```sh
-rustup target add x86_64-unknown-linux-musl
+reproduce <CONFIG_PATH>
 ```
 
-Then the portable REproduce binary, `reproduce`, can be created with the
-following command:
+* `<CONFIG_PATH>`: Path to the TOML configuration file.
 
-```sh
-cargo build --target x86_64-unknown-linux-musl --release
-```
+To support pcap parser for netflow v9, templates would be read from
+the environment variable `NETFLOW_TEMPLATES_PATH`.
 
-The compiled binary will be created in
-`target/x86_64-unknown-linux-musl/release`.
+## Configuration
 
-## Usage
+* Below is a detailed breakdown of all available configuration fields.
 
-To diplay the usage, type:
+### [Common]
```

Comment:
Is the term "common" used in the configuration? This doesn't sound ideal to me. I think "required" or "necessary" would be better because this section is not a shared part but a mandatory one.

Reply:
What do the brackets `[ ]` mean?